opensrp-chw/build.gradle

```diff
@@ -178,7 +178,7 @@ android {
             buildConfigField "String[]", "ALLOWED_LOCATION_LEVELS_DEBUG", '{"Clinic" , "CHSS" , "CHA"}'
             buildConfigField "String", 'DEFAULT_LOCATION_DEBUG', '"CHA"'
             buildConfigField "String", 'DEFAULT_LOCATION', '"CHA"'
-            buildConfigField "int", "DATABASE_VERSION", '6'
+            buildConfigField "int", "DATABASE_VERSION", '8'
```

Comment:
Is there any reasons why we are skipping db version 7?

Reply:
We had  migrations for version 7 already existing in the code. It was commented out and for some reason the database version was not changed to 7. So I added the new migration for version 8 and changed the DB version accordingly.